libs/core/src/lib/token/token.component.ts

```diff
@@ -18,42 +16,44 @@ import {
     selector: 'fd-token',
     templateUrl: './token.component.html',
     styleUrls: ['./token.component.scss'],
-    host: {
-        class: 'fd-token',
-        '[class.fd-token__disabled]': 'disabled',
-        'role': 'button'
-    },
     encapsulation: ViewEncapsulation.None,
     changeDetection: ChangeDetectionStrategy.OnPush
 })
 export class TokenComponent {
```

Comment:
Let's add there some onKeyDown  EventEmitter<KeyboardEvent>

Reply:
Would they not just use `<fd-token (keydown)="someFunction()">`?

Reply:
actually you are right, it's propagated upper.